cli/src/pcluster/constants.py

```diff
@@ -339,3 +339,6 @@ class Operation(Enum):
 IAM_ROLE_REGEX = "^arn:.*:role/"
 IAM_INSTANCE_PROFILE_REGEX = "^arn:.*:instance-profile/"
 IAM_POLICY_REGEX = "^arn:.*:policy/"
+
+# Section of Cloud-Init(cloud-config) which will be added for Disabling the Sudo access of Default user
+DISABLE_SUDO_ACCESS_FOR_DEFAULT_USER_CONFIG = {"system_info": {"default_user": {"sudo": ["ALL=(ALL) !ALL"]}}}
```

Comment:
*Not on this PR*, but an observation to be addressed in a follow up PR.
The `constants.py` file is getting bigger and bigger (more than 300 lines of constants) and I think we could start splitting it since most of the constants belongs to different categories. For this specific constant it could be put in a file `constants/cloud_init.py`